src/base/object_representation.h

```diff
@@ -85,167 +85,128 @@ struct setobj;
 #define OR_CHECK_FLOAT_OVERFLOW(i)         ((i) > FLT_MAX || (-(i)) > FLT_MAX)
 #define OR_CHECK_DOUBLE_OVERFLOW(i)        ((i) > DBL_MAX || (-(i)) > DBL_MAX)
 
+#if __WORDSIZE == 32
+#define OR_PTR_SIZE             4
+#define OR_PUT_PTR(ptr, val)    OR_PUT_INT ((ptr), (val))
+#define OR_GET_PTR(ptr)         OR_GET_INT ((ptr))
```

Comment:
OR_GET_INT is defined more belower from this. This could produce unintentional error at build time.

Reply:
Changed the location of the code.